docs/xml-config-file.rst

```diff
@@ -156,3 +164,77 @@ too.
         <publisher bridge_name="file" publisher_name="file_publisher"/>
         <transformation file="libfile.so" function="addTimestamp"/>
     </connector>
+
+There are several possible types of connectors depending of the kind of its participants.
+Each connector type will refer to :ref:`example`.
+
+RTPS Bridge
+^^^^^^^^^^^
+
+In this kind of connector, both participant are RTPS compliant, like *shapes_projection* and *shapes_stereo* in our example file.
+
+.. image:: RTPS-bridge.png
+    :align: center
+
+RTPS to Other protocol
```

Comment:
put xml example

Reply:
With "Each connector type will refer to :ref:example." I wanted to refer the bottom example, that, when clicked, opens the xml file.
Modified to "refer to the bottom :ref:example" for clearer reference.

Reply:
Added example xml.